protos/overlay/overlay.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+syntax = "proto3";
+
+import "duration.proto";
+
+// NodeTransport is an enum of possible transports for the overlay network
+enum NodeTransport {
+    TCP = 0;
+}
+
+// Overlay defines the interface for communication with the overlay network
+service Overlay {
+    // Lookup finds a nodes address from the network
+    rpc Lookup(LookupRequest) returns (LookupResponse);
+    // FindStorageNodes finds a list of nodes in the network that meet the specified request parameters
+    rpc FindStorageNodes(FindStorageNodesRequest) returns (FindStorageNodesResponse);
+}
+
+// LookupRequest is is request message for the lookup rpc call
+message LookupRequest {
+    string nodeID = 1;
+}
+
+// LookupResponse is is response message for the lookup rpc call
+message LookupResponse {
```

Comment:
what happens if reachable is false? nodeaddress is zero valued?

what happens if someone looks up a node id that has never existed?

Reply:
I was thinking errors would be returned for all of these cases instead of a response, including reachability. Are you suggesting we add fields for these cases ?